### DISCUSSION TITLE
part-ii/2018-49/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#157] [Question comments]
Part II 2018 Differential Geometry: Paper 1, Section II, I 

https://questions.tripos.org/part-ii/2018-49/

**Comment by eha1234** [comment]
A fun one.
<details><summary>Spoilers</summary>

(a) bookwork 

(b) Consider local coordinates, the tangent spaces are spanned by $\phi_u, \phi_v$. $\phi$ is smooth, hence has continuous partial derivatives. We may then write $w_i = a_i \phi_u(u_i,v_i) + b_i \phi_v(u_i,v_i)$. I then argued by contradiction. Suppose $w \notin T_pX$. $w_i$ converge, so in particular $a_i, b_i$ bounded. So $w_i$ can be made arbitrarily close to $T_pX$ as $i \rightarrow \infty$, giving the required contradiction. 

(c) Application of preimage theorem for $a \neq 0$. Dimension 2. For $a=0$, $X_a$ is a line, so clearly a manifold. Dimension is 1. 

(d) This is false. Take a sequence of $a_i$ tending to $0$, and a sequence $p_i = (a_i, 0, a_i, 1)$. This should give you a counterexample.  
</details>

Comments and corrections appreciated!
